src/open_inwoner/configurations/admin.py

```diff
@@ -339,6 +354,76 @@ def _get_column(props):
             ),
         )
 
+    def javascript_preview(self, obj):
+        """Show preview of the uploaded script"""
+        if obj.custom_javascript:
+            try:
+                obj.custom_javascript.seek(0)
+                content = obj.custom_javascript.read().decode("utf-8")
+                obj.custom_javascript.seek(0)
```

Comment:
Here I think we can use the contextmanager to avoid manual seeks:


```suggestion
                with obj.custom_javascript.open("r") as f:
                    content = f.read().decode("utf-8")
```